orchestrator-salt/src/main/resources/salt/pillar/top.sls

```diff
@@ -69,6 +69,7 @@ base:
     - match: grain
     - cloudera-manager.repo
     - cloudera-manager.communication
+    - postgresql.postgre
```

Comment:
Would we really like to have this postgre state for every CM agent node?
Basically it means every node of the deployment where CM is there...
What is you opinion @lacikaaa @keyki ?

Reply:
I understand your concern although I am a little unclear on the normal process for this. I simply needed the state/scripts present on the core nodes so they can be run on them and did not see an existing Salt role for just this.
Is this necessarily problematic? I do not see an issue with doing this, but am completely on board with keeping things tight and clean. If so, I would appreciate guidance on creating a new role and applying it to only the core nodes (note that this would need to be DL-scale specific as the medium duty node has multiple nodes marked as Core nodes in terms of instance type but only a few are actually named core nodes which are the ones I care about).

Reply:
Yes, I think this needs to be changed from the actual approach, because as I mentioned this change will apply the `postgre` state on every nodes of every deployment where CM exists, which means all nodes of every new Datahubs and Datalakes.
@keyki @lacikaaa  can we have your opinion about this, please?

Reply:
I think it is only the pillar for the passwords, etc, but maybe we don't need it in pillars either if we can understand the change better

Reply:
Oh it is just about Pillars, maybe it could be there

Reply:
This postgres pillar file contains all of the login info for Postgres, i.e. hostname, port, username, and password. Is this okay to include on these nodes?